tldr/daemon/startup.py

```
@@ -264,21 +266,61 @@ def start_daemon(project_path: str | Path, foreground: bool = False):
+                            print("Daemon already running")
+                            return
+
+                        # Get the connection info for display
+                        addr, port = daemon._get_connection_info()
+
+                        # Start detached process on Windows
+                        startupinfo = subprocess.STARTUPINFO()
+                        startupinfo.dwFlags |= subprocess.STARTF_USESHOWWINDOW
+                        startupinfo.wShowWindow = subprocess.SW_HIDE
+
+                        proc = subprocess.Popen(
```

Ruff (0.14.10)

`subprocess` call: check for execution of untrusted input

(S603)